package echo;

option cc_generic_services = true;
option py_generic_services = true;
option java_generic_services = true;

message EchoRequest
{
    required string payload = 1;
}

message EchoResponse
{
    required string payload = 1;
}

service EchoService
{
    rpc echo (EchoRequest) returns (EchoResponse);
}
